ldregistry/templates/structure/_footer.vm

```diff
@@ -24,6 +24,11 @@
           <span class="i18n">Design and development by</span>
           <a href="http://www.epimorphics.com">Epimorphics</a>.
         </p>
+        #if($registry.languageManager.useCookies)
+          <p>
+            This website uses cookies to store your language preferences.
```

Comment:
Might eventually need more here or a full a cookie approval dialog but I think that should be part of customising config-base for particular applications and happy to leave it as nice and simple here.